### DISCUSSION TITLE
Problem with imported BL-List to Lego Shop with Have Items?

### DISCUSSION THREAD

**Discussion by 81dirk81** [BrickTwo/BrickHunter#176] [Q&A]
With my last order I've got problems with a few parts, i think this was of what they where in BL. 
I used a want List with Items with "have" and "want". All have with zero where without problems. The others where dropped or strange values. 

eg some brick was 82 want, 38 have. In BH is 54 needed. But in Lego store there was thirtysomething. 

Is this an issue? Or missed I something, or the migration of the Lego Store?

**Comment by BrickTwo** [comment]
just tested it, seems to work fine so far. It shouldn't have something to do with the new PaB service.

- are you using the latest BrickHunter version? Should be verison 1.8.0
- is the setting "Subtract WantedList-Have from number" in the shopping cart->settings tab active? 

Can you maybe provide me your wanted list? you can send it to info@bricktwo.net

**Comment by 81dirk81** [comment]
Never notieced this switch, so it should be default "on". I mean it is up to date. Check this tommorow, send you my list.
